### DISCUSSION TITLE
Non-Searchable Properties: how to handle?

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#677] [Q&A]
From what I understand, the scheme for Orama is just the searchable properties, but then how do we deal with non-searchable properties? Do we just upload them into the Orama index, but just dno't reference them in the schema? For example, we are uploading our product database, which has a featured image (a URL). Obviously, we have no need to have the image be searchable. So we exclude that from the schema, but then Orama will still show it in the document result for a search? Thanks.

**Comment by micheleriva** [comment]
> Do we just upload them into the Orama index, but just don't reference them in the schema

Yes! That's correct. We wrote an article on this topic with more considerations: https://askorama.ai/blog/optimizing-orama-schema-optimization
